GitUI/CommandsDialogs/BrowseDialog/GitStatusMonitor.cs

```diff
@@ -28,7 +33,10 @@ public partial class GitStatusMonitor : IDisposable
         private bool _ignoredFilesAreStale;
         private string _gitPath;
         private string _submodulesPath;
+        //Timestamps to schedule status updates, limit the update interval dynamically
         private int _nextUpdateTime;
+        private int _prevUpdateTime;
+        private int _currUpdateInterval = MinUpdateInterval;
```

Comment:
We are not constrained in length for our variable names, so let's not unnecessary abbreviate: `_previousUpdateTime`, `_currentUpdateInterval`